windows/utils/win_install_online_updates.yml

```diff
@@ -46,10 +46,12 @@
   ansible.builtin.set_fact:
     win_online_updates_installed: true
   when:
+    - win_updates_install_result.failed is defined
+    - not win_updates_install_result.failed
```

Comment:
Will win_updates_install_result.failed be set to true if there's indeed some updates installation failure, not because of not applicable to OS?

Reply:
Don't know, we can handle all other situations when we meet.

Reply:
If `win_updates_install_result.failed` is true, then there might be something wrong during the installation, either task failed when not ignore errors, or `win_online_updates_installed` is not true.

Reply:
I suggest we can add two vars to identify different scenarios. One is for the scenario there's at lease one update installed. The other is for the number of installed updates is equal to the number of found updates.

Reply:
Changed to return "win_online_updates_succeed" and "win_online_updates_installed". Thanks.